src/components/Steps/__stories__/helpers.js

```diff
@@ -0,0 +1,14 @@
+export const modifiers = [
+  {
+    name: "preventOverflow",
```

Comment:
I think there is no need for this helper in the steps stories because the jump problem is not appear there

Reply:
Unfortunately, without it, the Tipseen would be jumping